docs/detections/rules-ui-manage.asciidoc

```diff
@@ -58,43 +58,46 @@ For {ml} rules, an indicator icon (image:images/rules-table-error-icon.png[Error
 [[edit-rules-settings]]
 === Modify existing rules settings
 
-You can edit an existing rule's settings, and can bulk edit settings for multiple rules at once.
+You can edit an existing custom rule's settings and can bulk edit settings for multiple rules at once. With the https://www.elastic.co/subscriptions/cloud[Enterprise] subscription, you can edit Elastic prebuilt rules and bulk modify them too.
```

Comment:
Let's mention that editing rule actions (notification and response actions) for prebuilt rules is allowed on any subscription.

Reply:
Added in [bb2e2cc](https://github.com/elastic/security-docs/pull/6563/commits/bb2e2cc0cb8a081dad416e0f58e86bb885234439).